store/sqlstore/channel_store.go

```diff
@@ -4255,3 +4256,51 @@ func (s SqlChannelStore) GetTopChannelsForUserSince(userID string, teamID string
 
 	return model.GetTopChannelListWithPagination(channels, limit), nil
 }
+
+func (s SqlChannelStore) PostCountsByDay(channelIDs []string, sinceUnixMillis int64) ([]*model.DailyPostCount, error) {
+	dailyPostCounts := make([]*model.DailyPostCount, 0)
+
+	postgresPropQuery := `AND (Posts.Props ->> 'from_bot' IS NULL OR Posts.Props ->> 'from_bot' = 'false')`
+	mySqlPropsQuery := `AND (JSON_EXTRACT(Posts.Props, '$.from_bot') IS NULL OR JSON_EXTRACT(Posts.Props, '$.from_bot') = 'false')`
+
+	query := `
+	SELECT
+		Posts.ChannelId AS channelid,
+		to_timestamp(Posts.CreateAt / 1000) :: date AS day,
+		count(Posts.Id) AS postcount
+	FROM
+		Posts
+		LEFT JOIN Channels on Posts.ChannelId = Channels.Id
+	WHERE
+		Posts.DeleteAt = 0
+		AND Posts.CreateAt > ?
+		AND Posts.Type = ''
```

Comment:
Regarding trying to not count system messages, this will miss any users' `/me` posts since those have a post type of `me`. We could check if `Posts.Type != 'system_%'` since most system message types are prefixed with that, but there's a few more recent ones like `add_bot_teams_channels` and `warn_metric_status` which seem like they might be system messages, but I honestly don't know what they're used for